templates/zerver/help/add-bots-or-integrations.md

```diff
@@ -0,0 +1,48 @@
+# Add a Bot or Integration
+
+Bots and Integrations are features that are accesible to all members of an Organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.
+
+    ![Settings Dropdown](/static/images/help/cog_dropdown_settings.png)
+
+3. The Settings window will replace your current one. Click on the **Your Bots**
+tab in the column on the left.
+
+    ![Bots Page](/static/images/help/bots_settings_page.png)
```

Comment:
This screenshot highlights the element that's not referred to in the comment above - looking at the screenshot alone, I'd expect to see "Your Bots" before clicking the tab. Could you make sure screenshot show the steps that are described above them?